src/main/java/org/swisspush/reststorage/RedisStorage.java

```diff
@@ -480,7 +475,7 @@ public void exec(final int executionCounter) {
                     String message = event.cause().getMessage();
                     if(message != null && message.startsWith("NOSCRIPT")) {
                         log.warn("get script couldn't be found, reload it");
-                        log.warn("amount the script got loaded: " + String.valueOf(executionCounter));
+                        log.warn("amount the script got loaded: " + executionCounter);

```

Comment:
Shouldn't we use placeholders anyway?
```java
log.warn("amount the script got loaded: {}", executionCounter);
```
(occurs multiple times)

Reply:
But is slf4j in use?